### ISSUE TITLE
ItemInfo reports disscontinued stats from wiki (Queen's_Escape)

### ISSUE THREAD

**Issue by beef682** [aRTy42/POE-ItemInfo#133]
Using ItemInfo in POE trade Macro 
I use Ctrl + C on Queen's Escape and ItemInfo reports 

Queen's Escape
Ornate Sword
Item Level:    59     Base Level:    27
Max Sockets:    6
Sockets:        G R G 
Ele DPS:      0.0     Chaos DPS:    0.0
Phys DPS:    58.1
Total DPS:   58.1
--------
+185 to Accuracy Rating                   
--------
Minions have increased maximum L…    10-15    ***
Minions have increased Movement …   80-100
Increased Skeleton Duration        150-200
Minions deal increased Damage        10-15        ***
+1 to maximum number of Zombies           
+1 to maximum number of Skeletons         
+1 to maximum number of Spectres          
25% increased Strength Requireme…

Minions have increased maximum Life and Minions deal increased Damage are shown pulling the discontinued item stats on the wiki, should read Minions have (30-40)% increased maximum Life and Minions deal (30-40)% increased Damage.

https://pathofexile.gamepedia.com/Queen's_Escape

just found on Queen's_Escape, dont know if found on other items.

ItemInfo ver  3.5.00
POETradeMacro ver  2.15.7
AutoHotkey ver  1.1.28.00
Deleted folder and fresh download/extract

**Comment by aRTy42** [comment]
I'm not maintaining ItemInfo as a standalone project at the moment, but the TradeMacro should still be more or less up to date. I just checked my data fetching script and it gets the new stats from the wiki. I'll ask Eruyome (from the TradeMacro) if he knows why the TradeMacro file is out of date.

**Comment by beef682** [comment]
yes, I posted the inquiry here as it is the parent code rather than on TradeMacro. :)
